public/src/components/channelManagement/helpers/shared.ts

```diff
@@ -99,8 +101,9 @@ export const APPLE_NEWS_EPIC_CONFIG: EpicEditorConfig = {
   allowMultipleVariants: false,
   allowCustomVariantSplit: false,
   allowContentTargeting: true,
-  allowLocationTargeting: false,
-  allowSupporterStatusTargeting: true,
+  allowLocationTargeting: true,
+  customRegions: [Region.UnitedStates, Region.AUDCountries],
+  allowSupporterStatusTargeting: false,
```

Comment:
This should have always been `false`, but the supporter status targeting was actually never shown if `allowLocationTargeting` was `false`